config.cc

```diff
@@ -2221,7 +2256,19 @@ QString getPidFileName() THROW_SPEC( exError )
 
 QString getHistoryFileName() THROW_SPEC( exError )
 {
+  #ifdef Q_OS_LINUX
```

Comment:
Preprocessor directives should be unindented and should not affect indentation of other code.

Reply:
Unindented as in indented on the same level as function `{` bracket, or not  indented at all? I ask because it's not entirely clear, there are various styles in this file, e.g. [this](https://github.com/goldendict/goldendict/blob/e92797a98a06f4fda934f6c02795821be69cbb2f/config.cc#L48) is the style I used, and [here](https://github.com/goldendict/goldendict/blob/e92797a98a06f4fda934f6c02795821be69cbb2f/config.cc#L2252) directive is indented, but it doesn't change indentation has not changed.

Reply:
Not indented at all. I am pretty sure this style is dominant in GoldenDict source code.